### ISSUE TITLE
Bug with date in Ubuntu 18.04

### ISSUE THREAD

**Issue by andresriancho** [prowler-cloud/prowler#245]
```
 1.3  [check13] Ensure credentials unused for 90 days or greater are disabled (Scored)
date: invalid date ‘+%s’
./include/os_detector: line 25: (1536721200 -  )/60/60/24: syntax error: operand expected (error token is ")/60/60/24")
./checks/check13: line 25: [: -gt: unary operator expected
```

**Comment by toniblyx** [comment]
Thanks @andresriancho. I think, that's related to the operating system. What OS are you using?

**Comment by andresriancho** [comment]
Ubuntu 18.04

**Comment by toniblyx** [comment]
Ok, I'm gonna check that out.

**Comment by toniblyx** [comment]
I'm trying to reproduce the error with the same OS version and I can't. Can you please share the output of this command? `bash -x ./prowler -c check13`, please fake aws account ID, etc.

**Comment by toniblyx** [comment]
For some reason this command doesn't work in your linux `date -d $DATE_TO_COMPARE +%s` 
https://github.com/toniblyx/prowler/blob/master/include/os_detector#L24 

If you can run `bash -x ./prowler -c check13` and send me the result I can see what else is going on.

**Comment by toniblyx** [comment]
This is finally fixed on PR #259
